apps/src/code-studio/pd/workshop_survey/WorkshopSurvey.jsx

```diff
@@ -47,6 +52,23 @@ export default class WorkshopSurvey extends FormController {
   /**
    * @override
    */
+  getRequiredFields() {
+    const extras = [];
+
+    if (this.props.isFirstSurvey) {
+      extras.push.apply(extras, this.props.demographicsRequiredFields);
+    }
+
+    if (this.props.subject === "Units 2 and 3: Web Development and Animations") {
```

Comment:
I'd love to see this string comparison either pulled into a method or comparing against something server-provided so we have to hardcode strings as little as possible